lib/iex/api/client.rb

```diff
@@ -22,12 +22,15 @@ class Client
       include Cloud::Connection
       include Cloud::Request
 
-      include Config
+      include Config::Client
+
+      attr_reader :logger
 
       def initialize(options = {})
-        Config::ATTRIBUTES.each do |key|
+        Config::Client::ATTRIBUTES.each do |key|
           send("#{key}=", options[key] || IEX::Api.config.send(key))
         end
+        @logger = Config::Logger.dup
```

Comment:
Not sure if this is the best approach but similar to the block above, we want to cache a local copy of the logger config this `Client` was instantiated with. Duplicating the module to enable the same dot access was the fastest way but another object could be build or a `Struct` used.